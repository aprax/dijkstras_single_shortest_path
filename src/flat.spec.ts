import findShortestPath from ".";
import type { Nodes } from './types';

const nodes: Nodes = new Map([
	['A', [['B', 1]]],
	['B', [['C', 2]]],
	['C', [['D', 3]]]
]);

it.only('should find the shortest path from A', () => {
	expect(findShortestPath('A', nodes)).toMatchSnapshot();
})