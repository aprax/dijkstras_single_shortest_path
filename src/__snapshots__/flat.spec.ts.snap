// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should find the shortest path from A 1`] = `
Map {
  "A" => Array [
    "A",
    0,
  ],
  "B" => Array [
    "A",
    1,
  ],
  "C" => Array [
    "B",
    3,
  ],
  "D" => Array [
    "C",
    5,
  ],
}
`;
